### PR TITLE
Add cloudflare-cdn-and-dns FAQ about Rocket Loader

### PR DESCRIPTION
**Pull request by primerg**

### CODE REVIEW

File: source/docs/articles/sites/cloudflare-cdn-and-dns.md [hunk 1]
```diff
@@ -54,3 +54,25 @@ In Drupal, no modules are required to use CloudFlare, but there is an optional [
 
 https://support.cloudflare.com/hc/en-us/articles/200169456-Why-is-JavaScript-or-jQuery-not-working-on-my-site-
 
+## FAQ
```
Comment (rachelwhitton) [inline source/docs/articles/sites/cloudflare-cdn-and-dns.md]: @nataliejeremy @primerg I suggest the following:
- Replace h2 **Known Limitations** in line 47 above with **Considerations**
- Remove the **FAQ** section added here, instead add an entry for **Rocket Loader** to match existing style and flow.

Thoughts? Thanks for the PR @primerg - much appreciated!

**Comment by nataliejeremy** [comment]
@rachelwhitton I think the FAQ section should stay, and the Rocket loader section needs a question like: Are there any issues with Rocket Loader?

**Comment by primerg** [comment]
@nataliejeremy @rachelwhitton Thank you for reviewing my first pull request. Please note that this is my first doc edit so do let me know what I need to do to improve this. Should I go ahead and retain FAQ while maintaining the existing style of Known limitations? Thanks!

**Comment by rachelwhitton** [comment]
@primerg awesome first PR! :tada: thanks again! 

I defer to @nataliejeremy on the styling suggestions

**Comment by nataliejeremy** [comment]
Thanks for the update @primerg 
Whenever you see anything that needs to be added, removed, or changed in a doc, feel free to submit a pull request by using the Contribute to Docs button within the doc itself. Don't worry about style or formatting, that's what we're here for so we'll take care of that. :) Thanks again!

**Comment by nataliejeremy** [comment]
Closing as new PR was created in #1022

**Pull request closed without merging**